go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -543,6 +568,14 @@ func addDefaultConntrackRules(nodeName, gwBridge, gwIntf string, stopChan chan s
 	}
 	nFlows++
 
+	// flush the shared gw bridge FDB to get rid of host/ovn entry that we may have picked up before all the flows
+	// were installed. This will force NORMAL action to behave like FLOOD for traffic sent to host/OVN MAC.
+	_, stderr, err = util.RunOVSAppctl("fdb/flush", gwBridge)
```

Comment:
should we do this periodically too?

Reply:
no this flush should only be needed after the initial flows are added. Since we have other flows that will always catch the traffic and send it out the physical port before the traffic can ever reach the NORMAL flow, there is no chance that the fdb can be updated by the host or OVN.